modules/pubmaticBidAdapter.js

```diff
@@ -18,6 +18,7 @@ const DEFAULT_HEIGHT = 0;
 const PREBID_NATIVE_HELP_LINK = 'http://prebid.org/dev-docs/show-native-ads.html';
 const PUBLICATION = 'pubmatic'; // Your publication on Blue Billywig, potentially with environment (e.g. publication.bbvms.com or publication.test.bbvms.com)
 const RENDERER_URL = 'https://pubmatic.bbvms.com/r/'.concat('$RENDERER', '.js'); // URL of the renderer application
+const VIDEO_PLACEMENT_MISSING_MSG = 'Video.Placement param missing';
```

Comment:
naming message related constants with "MSG_" at start will help in organizing code.

Reply:
Renamed above variable with "MSG_" at the start.